build.gradle

```diff
@@ -74,6 +74,7 @@ allprojects {
       trimTrailingWhitespace()
       endWithNewline()
       licenseHeaderFile "${rootDir}/gradle/spotless.java.license"
+      googleJavaFormat().aosp()
```

Comment:
We might want to recommend that everyone with one PRs add this to their build.gradle file and run the spotless step to minimize the conflicts.

Reply:
Yeah I agree. This is an annoying change, but luckily we only have to do it once.

Reply:
Is it ok if we put merging this on hold for a bit?

Reply:
Yep. In fact, jenkins/gradle is being a bit odd, so I'm gonna close this PR and I'll recreate it at another time.